<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GradientFillSymbolWidget</class>
 <widget class="QWidget" name="GradientFillSymbolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>175</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Central color</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ColorPickerButtont" name="centralColorBtn">
       <property name="text">
        <string>central color</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Surround color&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ColorPickerButtont" name="surroundColorBtn">
       <property name="text">
        <string>surround color</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="blendFactorsGroupBox">
     <property name="title">
      <string>Blend factors</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QSlider" name="blendFactor_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="blendFactor_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="blendPositionsGroupBox">
     <property name="title">
      <string>Blend positions</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QSlider" name="blendPosition_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="blendPosition_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="blendTriangularShapeGroupBox">
     <property name="title">
      <string>Blend triangular shape</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Focus: (0..1.0)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="blendTriangularBellShape_focus">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Scale: (0..1.0)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="blendTriangularBellShape_scale">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="blendBellShapeGroupBox">
     <property name="title">
      <string>Blend bell shape</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Focus: (0..1.0)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="blendBellShape_focus">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Scale: (0..1.0)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="blendBellShape_scale">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPickerButtont</class>
   <extends>QPushButton</extends>
   <header>ColorPickerButtont.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
